tests/testthat/test-compute_weighted_intensity_levels.R

```diff
@@ -0,0 +1,29 @@
+test_that("Can run the compute_weighted_intensity_levels fun without error", {
+
+  # Weibull fit
+  obs <- 10
+  season <- c("2018/2019", "2019/2020", "2020/2021")
+  season_num_rev <- rev(seq(from = 1, to = length(season)))
+  observations <- rep(stats::rnorm(10, obs), length(season))
+
+  peak_input <- tibble::tibble(
+    observation = observations,
+    weight = 0.8^rep(season_num_rev, each = obs)
+  )
```

Comment:
Just FYI. This looks like it is identical to the example.

When running `devtools::check()`, it also runs the examples and checks that no errors occur.
So you are essentially running this same test again here. 

Ideally, we should check a few more things during tests other than "does not fail", but it is hard in cases such as this where the is no "obvious" test to run on the function.